content/javascript/concepts/storage/terms/length.md

```diff
@@ -0,0 +1,22 @@
+# .length()
```

Comment:
```suggestion
```
Add the table here like this:
<img width="898" alt="image" src="https://github.com/Codecademy/docs/assets/94860622/dd0f7b19-30e6-4d25-bcb6-94113b007b63">
```
---
Title: 'Selection-Sort Algorithm'
Description: 'Returns a sorted array'
Subjects:
  - 'Computer Science'
Tags:
  - 'Algorithms'
  - 'Arithmetic'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---
```
You have to change the names and description according to the entry